frontendVfsUtils/src/main/java/com/virtuslab/gitmachete/frontend/vfsutils/GitVfsUtils.java

```diff
@@ -50,4 +53,13 @@ public static Path getMainDirectoryPath(GitRepository gitRepository) {
   public static Path getMacheteFilePath(GitRepository gitRepository) {
     return getGitDirectoryPath(gitRepository).resolve(MACHETE_FILE_NAME);
   }
+
+  /**
+   * @param filePath {@link Path} to file
+   * @return an option of {@link Long} stating for time of last modification in milliseconds
```

Comment:
For clarity: `milliseconds since the Unix epoch start`? if that's the intended semantics ofc